packages/shared/src/models/IRecipe.ts

```diff
@@ -25,6 +25,6 @@ export interface Recipe {
   repository: string;
   ref?: string;
   readme: string;
-  config?: string;
+  basedir?: string;
```

Comment:
```suggestion
  basedir: string;
```

could we also remove the `?` this field is not optional, I don't know why it has the ?.